vendor/re2.wake

```diff
@@ -18,7 +18,7 @@ from wake import _
 from gcc_wake import _
```

Comment:
You'll need to add a thing to remove `-std=c++11` from pkg-config flags in here as well